apps/explorer/src/ui/StatAmount.tsx

```diff
@@ -21,32 +22,19 @@ export function StatAmount({
     return (
         <div className="flex flex-col justify-start text-sui-grey-75 gap-2">
             <div className="text-sui-grey-100 flex flex-col items-baseline gap-2.5">
-                {date && (
-                    <div className="text-sui-grey-75">
-                        <Text variant="bodySmall" weight="semibold">
-                            {formatDate(date, [
-                                'month',
-                                'day',
-                                'year',
-                                'hour',
-                                'minute',
-                            ])}
-                        </Text>
-                    </div>
-                )}
-                <Heading as="h4" variant="heading4" weight="semibold">
-                    Amount
-                </Heading>
-
-                <div className="flex flex-row items-start gap-1">
-                    <Heading as="h2" variant="heading2" weight="bold">
-                        {amount}
+                {date && <DateCard date={date} />}
+                <div className="flex flex-col items-baseline gap-2.5">
+                    <Heading as="h4" variant="heading4" weight="semibold">
+                        Amount
                     </Heading>
-                    {currency && (
-                        <div className="text-sui-grey-80 ">
-                            <Text variant="bodySmall">{currency}</Text>
-                        </div>
-                    )}
+
+                    <div className="flex flex-row items-start gap-1">
```

Comment:
Curious why we need these classes

Reply:
I will remove it. This was before I separated the CoinBalance component.